myFirstWebsite/css/styles.css

```diff
@@ -96,55 +190,121 @@ body {
     font-weight: 500;
     font-size: 18px;
     line-height: 24px;
-    color: #FFFFFF;
+    color: var(--search-form-text-color);
     margin-right: 60px;
     padding-left: 43px;
     padding-bottom: 22px;
+    /* https://stackoverflow.com/questions/917610/put-icon-inside-input-element-in-a-form */
```

Comment:
It will be nice if you remove unnecessary code

Reply:
+